yksgi/src/lib.rs

```diff
@@ -414,6 +415,8 @@ impl SGInterp {
             } else {
                 todo!("{:?}", ret_ty.as_str());
             };
+            drop(ffi_arg_vals);
```

Comment:
I think a comment explaining why `drop` is needed would be useful (I had to stop and think!).

Reply:
Is 4d5b199 OK?